test/pkg/environment/common/expectations.go

```diff
@@ -294,6 +294,19 @@ func (env *Environment) EventuallyExpectTerminatingWithTimeout(timeout time.Dura
 	}).WithTimeout(timeout).Should(Succeed())
 }
 
+func (env *Environment) eventuallyExpectNoLeakedKubeNodeLease() {
+	GinkgoHelper()
+	Eventually(func(g Gomega) {
```

Comment:
What's the reason that this needs to be an eventually statement?

Reply:
Agreed, unnecessary here.